datashare-app/src/test/java/org/icij/datashare/tasks/TaskManagerAmqpTest.java

```diff
@@ -29,8 +28,9 @@ public class TaskManagerAmqpTest {
     @ClassRule static public AmqpServerRule qpid = new AmqpServerRule(5672);
     TaskManagerAmqp taskManager;
     TaskSupplierAmqp taskSupplier;
+    CountDownLatch nextMessage;
 
-    @Test
+    @Test(timeout = 2000)
```

Comment:
Why do we need to increase the timeout here?

Reply:
because we've seen that the test may block indefinitely.

Reply:
Ah ok, make sense!